### PR TITLE
[VL][DOC] Add uniffle doc

### PR DESCRIPTION
**Pull request by summaryzb**
## What changes were proposed in this pull request?
Add uniffle doc

### CODE REVIEW

**Comment by summaryzb** [comment]
cc @jackylee-ch

File: docs/get-started/Velox.md [hunk 1]
```diff
@@ -264,6 +264,44 @@ spark.celeborn.storage.hdfs.dir hdfs://<namenode>/celeborn
 spark.dynamicAllocation.enabled false
 ```
 
+## Uniffle support
+
+Uniffle with velox backend supports [Uniffle](https://github.com/apache/incubator-uniffle) as remote shuffle service. Currently, the supported Uniffle versions are `0.8.0` and `0.9.0`.
```
Comment (jackylee-ch) [inline docs/get-started/Velox.md:269]: The supported for Uniffle version `0.9.0` is still working in #6484 , It is better to remove the `0.9.0` if it cannot be merged in `1.2.0`.

**Comment by jackylee-ch** [comment]
Also cc @weiting-chen

File: docs/get-started/Velox.md (hunk 1 above)
Comment (summaryzb) [reply docs/get-started/Velox.md]: ok

**Review by jackylee-ch — APPROVED** [review]
LGTM. Thanks

**Comment by PHILO-HE** [comment]
@summaryzb, please create another pr to port it to branch-1.2. Thanks! cc @weiting-chen

**Pull request merged**